mmdet/apis/inference.py

```diff
@@ -10,7 +10,8 @@
 def _prepare_data(img, img_transform, cfg, device):
     ori_shape = img.shape
     img, img_shape, pad_shape, scale_factor = img_transform(
-        img, scale=cfg.data.test.img_scale)
+        img, scale=cfg.data.test.img_scale, 
```

Comment:
You have a trailing whitespace here, which is unexpected and fails the flake8 linting.

If you use yapf to auto-format the code, it should look like this.
```python
img, img_shape, pad_shape, scale_factor = img_transform(
    img,
    scale=cfg.data.test.img_scale,
    keep_ratio=cfg.data.test.get('resize_keep_ratio', True))
```